src/transaction_manager/transaction_manager.cpp

```diff
@@ -229,7 +229,7 @@ void TransactionManager::verifyQueuedTrxs() {
       auto status = db_->getTransactionStatus(hash);
       if (status == TransactionStatus::in_queue_unverified) {
         db_->saveTransactionStatus(hash, TransactionStatus::in_queue_verified);
-
+        db_->saveTransaction(*item.second, true);
```

Comment:
so we have to save the same tx twice into db, for the first time when saving unverified tx and second time here ?

Reply:
no it will be just this one, this will just update byte value of tx in db